fail2ban/server/server.py

```diff
@@ -402,8 +434,14 @@ def getLogLevel(self):
 	# @param target the logging target
 	
 	def setLogTarget(self, target):
-		try:
-			self.__loggingLock.acquire()
+		with self.__loggingLock:
+			# don't set new handlers if already the same
+			# or if "INHERITED" (foreground worker of the test cases, to prevent stop logging):
+			if self.__logTarget == target:
+				return True
+			if target == "INHERITED":
```

Comment:
INHERITED is just for the test cases (it tests now starting of the server, and server should log in the same test environment, so possible to capture and analyse it hereafter)